config/sidekiq_scheduler.yml

```diff
@@ -214,3 +214,8 @@ Veteran::VSOReloader:
   cron: "0 2 * * * America/New_York"
   class: Veteran::VSOReloader
   description: Pulls down current list of VSOs and updates accordingly 
+
+CovidVaccine::ScheduledBatchJob:
+  cron: "0 6-19 * * MON-FRI America/New_York"
```

Comment:
Inferring from a previous convo that if we want to change this value to hourly or some other cadence, we can do so without a full vets-api deploy (i.e. just re-deploying the last release)?

Reply:
No...because this file, while technically a config file, isn't overriden in the devops repo the way the main settings.yml file is.

Reply:
Caveat: I have been known to mess up cron syntax before 🙈

Reply:
Touche. Correct phrase would have been "every hour, regardless of hour of day or day of week" maybe.

Reply:
https://crontab.guru/#0_6-19_*_*_MON-FRI seems to like it, fwiw.